stone/api.py

```diff
@@ -96,7 +96,7 @@ class ApiNamespace(object):
     """
 
     def __init__(self, name):
-        # type: (str) -> None
+        # type: (typing.Text) -> None
```

Comment:
@braincore, as a follow-up to our out-of-band discussion, `typing.Text` was introduced in 3.5.2, so [we should be okay](/wittekm/stone/blob/3cf59644e4ed53f079ca33da29f3733158bc4fa5/tox.ini#L29). 👌